lib/backend/repositories/auth_repo.dart

```diff
@@ -96,5 +97,6 @@ class AuthRepo {
     }
     logger.logInfo('Logging out');
     await authService.logOut();
+    await cache.wipe();
```

Comment:
The LocalCache was not passed on purpose. dbCache.wipe() is already being called here:
![image](https://user-images.githubusercontent.com/16854239/87220797-7cbc8980-c367-11ea-97b3-60fdd75cb108.png)